lib/steps.js

```diff
@@ -80,15 +96,15 @@ Before(async (scenario) => {
 Given(/I start the interview at "([^"]+)"(?: in lang "([^"]+)")?/, async (file_name, language) => {  // √
   await scope.load( scope, file_name );
 
-  scope.filename_suffix = `_${scope.scenario_name}`;
-  if ( language ) { scope.filename_suffix = `${scope.filename_suffix}_${ language }`; }
+  // This won't be needed after tags get added in
```

Comment:
Are you referring to https://github.com/plocket/docassemble-cucumber/pull/96/files?

Reply:
Thanks! I made this branch before the tags branch was merged in. The tags already add the language name, so I can remove this.